src/index.html

```diff
@@ -11,6 +11,49 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <div class="search">
+      <form
+        class="search__form"
+        action="http://localhost:8080/"
+        method="post"
+        data-qa="big"
+      >
+        <input
+          class="search__input search__input--big"
+          type="text"
+          name="search"
+          id="search"
+          data-qa="keypress"
+          placeholder="Try &quot;Los Angeles&quot;"
+        >
+        <img
+          class="search__icon"
+          src="images/Search.svg"
+          alt="search"
+        >
+      </form>
+    </div>
+
+    <div class="search">
+      <form
+        class="search__form"
+        action="http://localhost:8080/"
```

Comment:
- if you want to add some plug for actions is's better to use # instead of hardcoded host 
```suggestion
        action="#"
```